src/main/java/com/compstore/mapper/PCMapper.java

```diff
@@ -27,6 +28,12 @@ public interface PCMapper {
     @Mapping(target = "operatingSystem", ignore = true)
     PCEntity toEntity(PCCreateRequestDTO pcCreateRequest);
 
+    @Mapping(target = "id", ignore = true)
+    @Mapping(target = "processorBrand", ignore = true)
+    @Mapping(target = "graphicsCardBrand", ignore = true)
+    @Mapping(target = "operatingSystem", ignore = true)
+    void toEntity(PCCreateRequestDTO pcUpdateRequest, @MappingTarget PCEntity existingPC);
```

Comment:
Adnotacja @MappingTarget w MapStruct jest używana w celu określenia celu mapowania. Jest to szczególnie użyteczne w przypadku aktualizacji istniejącego obiektu, a nie tworzenia nowego.